Dockerfile

```diff
@@ -1,4 +1,13 @@
-FROM node:9-onbuild
+FROM node:9
+
+RUN mkdir -p /usr/src/app
+WORKDIR /usr/src/app
+
+ARG NODE_ENV
+ENV NODE_ENV $NODE_ENV
```

Comment:
Ah, I see `NODE_ENV` is specified on line 12 already, so strictly speaking we can delete this line. It won't do any harm leaving it in though, so up to you.